File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1113.rb
while(true) do
    ns = gets.chomp().split(' ')
    x = ns[0].to_i
    y = ns[1].to_i
    if(x==y) then
        break
    end
    if(x>y) then
        puts("Decrescente")
    else
        puts("Crescente")
    end
end

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1113.c
#include <stdio.h>

int main(){
    int x, y;
    while(1){
        scanf("%d %d", &x, &y);
        if(x==y){
            break;
        }
        if(x>y){
            puts("Decrescente");
        } else{
            puts("Crescente");
        }
    }

    return 0;
}